eogrow/pipelines/export_maps.py

```diff
@@ -74,6 +74,13 @@ class Schema(Pipeline.Schema):
                 " With this parameter you force to always make copies."
             ),
         )
+        temporary_files_suffix: str = Field(
```

Comment:
this name is the only thing that bothers me in the MR, but I also don't have a better suggestion. The name looks scary, but if you simplify it, we increase risk of misinterpretation.

Perhaps `patch_suffix` or something could also work?

Can you describe the issue again? IIRC you mentioned it was an issue when running for different quarters?

Reply:
`clash_prevention_suffix` :P

The case we had was downloading hella data (mosaicking and all that jazz). So we had to separate the data into quarters for 3 years, or the batch API wouldn't be able to handle it. And then we ingested it as a BYOC.

For each quarter we ran the same pipeline, which extracted the tiff into a map named `BANDS`. So if i tried to run two pipelines in parallel, the exported tiffs had the same names and were overriding eachother

Reply:
How about a "simple" `additional_suffix` or perhaps more descriptive `distinctive_suffix`?

Reply:
I would like the name to convey that the suffix will NOT be part of the final map. Was also considering `partial_results_suffix`.

Reply:
How about `interim_results_suffix`

Reply:
> I would like the name to convey that the suffix will NOT be part of the final map.

This is my main issue with this, if it's not a part of the final map, why even bother the user with this in the first place, even if it's just meant for those last 5%.

TBH, I'm fine with any of the choice above (perhaps also `working_suffix`), if this is the only way to go ahead. I was just trying to see if there is a way of automatizing this.

For example, what are the differences between the configs mentioned by @zigaLuksic above? IIRC, it's the variables. Perhaps all available variables could be automatically used in the temporary names? But then again, the used isn't forced to use variables, they could just create multiple configs with different paths, so we're back to square 1

Or perhaps hex-string of the config? it should remain the same for the same configs and would be easy to clean up in case of any failure.

Reply:
using other parameters automatically gives you 'uniqueness of runs', because if it fails something was probably wrong with the config.
We can have one of:
1. auto-unique (enough) suffixes, cannot do cleanup with subsequent runs
2. the user adds a suffix parameter if needed (it has 'no suffix' as default, no reason to set it, usually one only sets around half of params anyway)
3. no suffixes, causing issues with parallelization in temporally-split cases.

Reply:
>  because if it fails something was probably wrong with the config.

Good point, yes.

Then let's go with item 2. I would vote for `interim_suffix` or `working_suffix`. It's different enough to not assume what it is (hopefully), and then you have the description for what it really is.

Reply:
done, also stated that it does not affect results in the first sentence of description